webcompat/static/js/lib/wizard/step1.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
So, maybe an annoying request... but could we write a small comment block at the top of each step module that explains what it does?

```
/* step1.js - allows the user to enter a URl and navigate to the next step */

(or something less crappy, just something to give some context as we revisit these in the future)

Reply:
Yeah, good point :)